index.html

```diff
@@ -0,0 +1,227 @@
+            <p class="home-page-main-program-paragraph"> Try creating your own work using open source license rather than just watching at it.</p>
+          </div>
+          <div class="home-page-main-program-5">
+            <img src="image/program_icon_05.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-5-oratory">Ignite</h2>
+            <p class="home-page-main-program-paragraph"> get opportunities to network with CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+
+          <!--    button to submit    -->
+
+        <button class="home-page-info-joinNowBtn">GET YOUR PASS</button>
+
+      </div>
+    </section>
+      
+      <!--    speaker me page   -->
+    
+    <section class="speaker-page">
+      <h1 class="speaker-page-h1">Featured Speaker</h1>
+      <div class="speaker-page-h1-div"></div>
+      <div class="speaker-page-div">
+      </div>
+        <!--    add here with js   -->
+    </section>
+    
+    <!--    about me page   -->
+
+    <section class="about-page" id="about-page">
+
+      <!--    titles    -->
+
+      <div class="about-page-tittles">
+        <h2 class="about-page-h2">"Security is the future"</h2>
+        <h1 class="about-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+      </div>
+
+      <!--    description of the event    -->
+
+      <div class="about-page-text">
+        <p class="about-page-text-p">Blockchain Summit is back, 
+          one of the most popular and important crypto events worldwide,
+           aimed at market leaders, Web3 sector entrepreneurs, investors,
+            and developers. To attend with a 15% discount,
+             use the promotional code BCS2023.
+        </p>
+      </div>
+      
+      <!--    contact info    -->
+
+      <div class="about-page-info">
+        <h2 class="about-page-info-date">Please contact us per Email for nay further questions about Block-chain first TECH SUMMIT 2023:</h2>
+        <h3 class="about-page-info-place">mail: cryptoconferences@gmail.com & instagram: cryptOk</h3>
+      </div>
+      
+    </section>
+
+    <section class="about-page-past-content">
+      <!--  titles of page past  -->
+      <div class="about-page-past">
+          <h1 class="about-page-past-h1">Creative Commons Block-chain TECH SUMMIT 2023 Logo</h1>
+          <div class="about-page-past-border"></div>
+          <h2 class="about-page-past-h2">The logo Block-chain TECH SUMMIT 2023 was decided through 
+            the logo competition from 8, June to 7, July.</h2>
+      </div>
+      <!--  my logo-->  
+      <div class="about-page-past-logo">
+        <img src="image/logo-no-circle.png" alt="" width="100px">
+        <h1 class="about-page-past-logo-text">Libra</h1>
+      </div>
+      <!--    --> 
+      <div class="about-page-past">
+        <h1 class="about-page-past-h1">See the past Block-chain TECH SUMMIT 2023</h1>
```

Comment:
- [ ] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍